src/components/NcAppContent/NcAppContent.vue

```diff
@@ -202,6 +209,16 @@ export default {
 			type: String,
 			default: null,
 		},
+		/**
+		 * Switch between current layout(vertical split) and no split layout. No split layout shows only 2 columns, navigation+list or navigation+content
+		 */
```

Comment:
Navigation is not a part of this component. So `no-split` is 1-column.

```suggestion
		/**
		 * Content layout used when there is a list together with content:
		 * - `vertical-split` - a 2-column layout with list and default content separated vertically
		 * - `no-split` - a single column layout; List is shown when `showDetails` is `false`, otherwise the default slot content is shown with a back button to return to the list.
		 * On mobile screen `no-split` layout is forced.
		 */
```